Moonscraper Chart Editor/Assets/Scripts/Game/Controllers/Song Objects/DrumRollController.cs

```diff
@@ -214,6 +217,25 @@ void SetLanePosition(GameObject laneVisuals, int drumRollNoteIndex)
             scale.y = yMax - yMin;
             laneVisuals.transform.localScale = scale;
 
+            Skin customSkin = SkinManager.Instance.currentSkin;
+            int colorIndex = (int)pad;
+            if (colorIndex < customSkin.sustain_mats.Length)
+            {
+                var sprite = laneVisuals.GetComponent<SpriteRenderer>();
+                Color color;
+                if (customSkin.sustain_mats[colorIndex])
+                {
+                    color = customSkin.sustain_mats[colorIndex].color;
+                }
+                else
+                {
+                    color = resources.sustainColours[colorIndex].color;
+                }
+
+                color.a = 0.5f;
```

Comment:
This should be in constant near the top of the file rather than a magic number. Even better should be a serializable field so that the alpha level can be edited in the inspector instead.